### ISSUE TITLE
Failure to launch

### ISSUE THREAD

**Issue by ErinWeisbart** [CellProfiler/CellProfiler#4340]
**Describe the bug**
Failure to launch

```5:26:56 PM: Error: Unable to set default locale: 'unsupported locale setting'
5:26:56 PM: Error: Unable to set default locale: 'unsupported locale setting'
5:26:56 PM: Error: Unable to set default locale: 'unsupported locale setting'
5:26:56 PM: Error: Unable to set default locale: 'unsupported locale setting'
5:26:57 PM: Error: Unable to set default locale: 'unsupported locale setting'
Python rejected the system locale detected by WX ('en_GB').
This shouldn't cause problems, but please let us know if you encounter errors.
Traceback (most recent call last):
  File "c:\users\administrator\documents\github\cellprofiler\cellprofiler\cellprofiler\gui\app.py", line 69, in OnInit
    from .cpframe import CPFrame
  File "c:\users\administrator\documents\github\cellprofiler\cellprofiler\cellprofiler\gui\cpframe.py", line 26, in <module>
    from ._workspace_model import Workspace
  File "c:\users\administrator\documents\github\cellprofiler\cellprofiler\cellprofiler\gui\_workspace_model.py", line 8, in <module>
    import cellprofiler.gui.figure
  File "c:\users\administrator\documents\github\cellprofiler\cellprofiler\cellprofiler\gui\figure\__init__.py", line 1, in <module>
    from ._figure import Figure
  File "c:\users\administrator\documents\github\cellprofiler\cellprofiler\cellprofiler\gui\figure\_figure.py", line 8, in <module>
    import centrosome.cpmorphology
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\centrosome\cpmorphology.py", line 12, in <module>
    from ._cpmorphology2 import skeletonize_loop, table_lookup_index
  File "centrosome\_cpmorphology2.pyx", line 1, in init centrosome._cpmorphology2
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
OnInit returned false, exiting...
```
```
boto3==1.16.63
botocore==1.19.63
-e git+https://github.com/CellProfiler/CellProfiler.git@91ff02034e0cc400d72c56cba85432388f057da5#egg=CellProfiler
cellprofiler-core==4.0.7
centrosome==1.2.0
certifi==2020.12.5
chardet==4.0.0
cycler==0.10.0
decorator==4.4.2
deprecation==2.1.0
docutils==0.15.2
future==0.18.2
h5py==2.10.0
idna==2.10
imageio==2.9.0
inflect==5.0.2
Jinja2==2.11.3
jmespath==0.10.0
joblib==1.0.0
kiwisolver==1.3.1
mahotas==1.4.11
MarkupSafe==1.1.1
matplotlib==3.1.3
mysqlclient==1.4.6
networkx==2.5
numpy==1.19.3
packaging==20.9
Pillow==8.1.0
prokaryote==2.4.2
psutil==5.8.0
pyparsing==2.4.7
python-bioformats==4.0.0
python-dateutil==2.8.1
python-javabridge==4.0.0
PyWavelets==1.1.1
pyzmq==18.0.1
requests==2.25.1
s3transfer==0.3.4
scikit-image==0.18.1
scikit-learn==0.24.1
scipy==1.6.0
sentry-sdk==0.18.0
six==1.15.0
threadpoolctl==2.1.0
tifffile==2021.1.14
urllib3==1.26.3
wxPython==4.1.1
```

**Desktop (please complete the following information):**
 - OS: Windows Server 2016
 - CellProfiler Version: 4
 - source

Uninstalled and reinstalled centrosome a couple times but it didn't help.

**Comment by bethac07** [comment]
Python 3.8.7, IIRC

**Comment by DavidStirling** [comment]
Fixed in 4.1.3
